lib/mail_room/mailbox.rb

```diff
@@ -84,6 +86,7 @@ def deliver(message)
       body = message.attr['RFC822']
       return true unless body
 
+      MailRoom.structured_logger.info("#{context} Delivering #{message.attr['UID']} through #{delivery.class.name} (#{message.attr['RFC822.SIZE']} bytes)")
```

Comment:
Same for the info here. I think it would be useful to have the `context`, `UID`, delivery class, and size as JSON attributes so that we can filter / report on these.